Dockerfile

```diff
@@ -24,10 +24,10 @@ COPY --from=builder /build/libs/ /usr/local/spark/jars/
 
 ENV HOME=
 WORKDIR /deps
-COPY requirements.txt /deps/
+COPY local_dev /deps/spark_notebook_base
```

Comment:
should we move pyproject.toml + uv.lock out of local_dev? They are no longer meant to be used for local development and are now required by the service.